k8s/prod/common/ccd/case-activity-api.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccd-case-activity-api
+  namespace: ccd
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.nodejs: glob:prod-*
+spec:
+  releaseName: ccd-case-activity-api
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ccd-case-activity-api
+    version: 1.0.4
+  values:
+    nodejs:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/ccd/ccd-case-activity-api:latest
+      ingressHost: gateway.ccd.platform.hmcts.net
```

Comment:
@MisterSK I don't think the ingress is correct.  Could you verify if this ingress is for this app?